boms/pom.xml

```diff
@@ -1,73 +1,34 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+               http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud</groupId>
-  <artifactId>zero-pom-test</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <groupId>com.google.cloud.tools.opensource</groupId>
+  <artifactId>cloud-tools-opensource-parent</artifactId>
   <packaging>pom</packaging>
-
-  <name>Google Cloud Platform Supported Libraries</name>
-  <description>
-    A compatible set of Google Cloud open source libraries.
-  </description>
-
+  <version>1.0-SNAPSHOT</version>
+  <name>BOM Parent</name>
+  <url>https://github.com/GoogleCloudPlatform/cloud-opensource-java/boms</url>
   <organization>
-    <name>Google LLC</name>
-    <url>https://cloud.google.com</url>
+    <name>Google LLC.</name>
+    <url>http://www.google.com</url>
   </organization>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
   <licenses>
     <license>
       <name>The Apache License, Version 2.0</name>
       <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
     </license>
   </licenses>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <google.cloud.java.ga.version>1.38.0</google.cloud.java.ga.version>
-  </properties>
-
-  <dependencies>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>20.0</version>
-    </dependency>
-
-    <!-- Google Cloud Java APIs -->
-    <dependency>
-      <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-storage</artifactId>
-      <version>${google.cloud.java.ga.version}</version>
-    </dependency>
-  </dependencies>
-
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <version>3.0.0-M2</version>
-        <executions>
-          <execution>
-            <id>enforce</id>
-            <goals>
-              <goal>enforce</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <rules>
-            <dependencyConvergence/>
-          </rules>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <modules>
+    <module>zero-bom</module>
+    <module>upperBoundsCheck</module>
```

Comment:
Should these use a consistent naming scheme? lower-hyphenated vs lowerCamelCase

Specifically, should the directory names match the artifact names?

Reply:
I don't know if directory names should match artifact names, but they certainly shouldn't be the same. Fixed. 

Otherwise, I'm not aware of any conventions on artifact IDs.